infrastructure/main.tf

```diff
@@ -32,37 +32,38 @@ module "ccpay-bulkscanning-payment-database" {
   sku_name = "GP_Gen5_2"
   sku_tier = "GeneralPurpose"
   common_tags = "${var.common_tags}"
+  postgresql_version = "${var.postgresql_version}"
```

Comment:
it this variable used internally? or are we using that? can you please let us know.